backend/src/scraper/consumer_reports_scraper.py

```diff
@@ -0,0 +1,54 @@
+from os import environ
+import requests
+from typing import Dict
+import difflib
+
+def scrape_expert_reviews(q: str, **kwargs) -> Dict:
+    short_title = ' '.join(q.split()[:5])
+    upc = kwargs.get('upc', '')
+    walmart_id = kwargs.get('walmart_id', '')
+
+    headers = {
+        "X-RapidAPI-Key": environ.get("RAPIDAPI_API_KEY"),
+        "X-RapidAPI-Host": "consumer-reports.p.rapidapi.com"
+    }
+
+    # Get all products in camera category.
+    all_prod_url = "https://consumer-reports.p.rapidapi.com/products/list"
+    all_prod_querystring = {"productGroupId":"28960","page":"0","size":"100"}
+    all_prod_response = requests.get(all_prod_url, headers=headers, params=all_prod_querystring)
+    all_prod_results = all_prod_response.json()
+
+    # Create name to id mapping.
+    name_id = {}
+    for prod in all_prod_results.get('content'):
+        name_id[prod.get('modelName')] = prod.get('_id')
+
+    # Return id of product name that most closely matches shortened q.
+    id = max(name_id.items(), key = lambda x: difflib.SequenceMatcher(None, short_title, x[0]).ratio())[1]
```

Comment:
Is there a chance this returns nothing? Or is it guaranteed to return something even if it's not a close match?

Reply:
It's guaranteed to return something. A comparison score will always be returned (even if it's 0).

Reply:
Comparison score? Doesn't this line of code return an `id`?

Reply:
Oh yeah, but there will always be items in name_id since this comes from a general search of all camera products. 
An id will always be returned.